lib/Mojolicious/Plugin/AssetPack/Pipe/Export.pm

```diff
@@ -0,0 +1,92 @@
+
+package Mojolicious::Plugin::AssetPack::Pipe::Export;
+
+use Mojo::Base 'Mojolicious::Plugin::AssetPack::Pipe';
+use Mojo::Util 'spurt';
+
+use File::Basename 'dirname';
+use File::Path 'make_path';
+use File::Spec;
+
+has 'export_dir';
+has 'use_checksum_subdir' => 1;
+
+sub process {
```

Comment:
You should run perltidy on the code. There's a .perltidyrc in the root of the project which will format the code.